components/clp-py-utils/clp_py_utils/initialize-clp-metadata-db.py

```diff
@@ -23,88 +31,36 @@
-                    `size` BIGINT NOT NULL,
-                    `creator_id` VARCHAR(64) NOT NULL,
-                    `creation_ix` INT NOT NULL,
-                    KEY `archives_creation_order` (`creator_id`,`creation_ix`) USING BTREE,
-                    UNIQUE KEY `archive_id` (`id`) USING BTREE,
-                    PRIMARY KEY (`pagination_id`)
-                )
-                """
-            )
-
-            metadata_db_cursor.execute(
-                f"""
-                CREATE TABLE IF NOT EXISTS `{table_prefix}tags` (
-                    `tag_id` INT unsigned NOT NULL AUTO_INCREMENT,
-                    `tag_name` VARCHAR(255) NOT NULL,
-                    UNIQUE KEY (`tag_name`) USING BTREE,
-                    PRIMARY KEY (`tag_id`)
-                )
-                """
-            )
-
-            metadata_db_cursor.execute(
-                f"""
-                CREATE TABLE IF NOT EXISTS `{table_prefix}archive_tags` (
-                    `archive_id` VARCHAR(64) NOT NULL,
-                    `tag_id` INT unsigned NOT NULL,
-                    PRIMARY KEY (`archive_id`,`tag_id`),
-                    FOREIGN KEY (`archive_id`) REFERENCES `{table_prefix}archives` (`id`),
-                    FOREIGN KEY (`tag_id`) REFERENCES `{table_prefix}tags` (`tag_id`)
-                )
-                """
-            )
-
-            metadata_db_cursor.execute(
-                f"""
-                CREATE TABLE IF NOT EXISTS `{table_prefix}files` (
-                    `id` VARCHAR(64) NOT NULL,
-                    `orig_file_id` VARCHAR(64) NOT NULL,
-                    `path` VARCHAR(12288) NOT NULL,
-                    `begin_timestamp` BIGINT NOT NULL,
-                    `end_timestamp` BIGINT NOT NULL,
-                    `num_uncompressed_bytes` BIGINT NOT NULL,
-                    `begin_message_ix` BIGINT NOT NULL,
-                    `num_messages` BIGINT NOT NULL,
-                    `archive_id` VARCHAR(64) NOT NULL,
-                    KEY `files_path` (path(768)) USING BTREE,
-                    KEY `files_archive_id` (`archive_id`) USING BTREE,
-                    PRIMARY KEY (`id`)
-                ) ROW_FORMAT=DYNAMIC
-                """
-            )
-
-            metadata_db_cursor.execute(
-                f"""
-                CREATE TABLE IF NOT EXISTS `{table_prefix}column_metadata_default` (
-                    `name` VARCHAR(512) NOT NULL,
-                    `type` TINYINT NOT NULL,
-                    PRIMARY KEY (`name`, `type`)
-                )
-                """
-            )
-
+            # TODO: After the dataset feature is implemented:
+            #       1. Populate the `clp-datasets` table with the name and path for the `default`
+            #          dataset.
+            #       2. Create a set of metadata tables for the `default` dataset.
+            #       3. Remove the default db tables for CLP_S.
```

Comment:
```suggestion
            # TODO: After the dataset feature is fully implemented, for clp-json:
            # 1. Populate the datasets table with the name and path for the "default" dataset.
            # 2. Change the metadata tables to be specific to the "default" dataset.
```

I think that's semantically equivalent?

Reply:
yeah